omnibus/package-scripts/agent/postinst

```diff
@@ -90,6 +90,28 @@ if [ "$DISTRIBUTION" != "Darwin" ]; then
             echo "[ WARNING ]\tCannot detect a supported init system. The datadog-agent package only provides service files for systemd, upstart and sysvinit."
         fi
 
+        # Setup SELinux policy and label if SELinux detected on the host
+        if command -v semodule >/dev/null 2>&1 && [ -f "$INSTALL_DIR/embedded/bin/system-probe" ]; then
+            echo "Loading SELinux policy module for datadog-agent."
+            semodule -v -i $CONFIG_DIR/selinux/system_probe_policy.pp >/dev/null 2>&1
+            if [ "$?" != "0" ]; then
+                echo "The SELinux policy module for datadog-agent is not compatible with this host."
+                echo "Please install or update the selinux-policy-default package and then reinstall the datadog-agent package to configure SELinux for the Datadog Agent."
+                echo "Skipping SELinux configuration."
+            else
+                echo "Labeling SELinux type for the system-probe binary."
+                if command -v semanage >/dev/null 2>&1 && command -v restorecon >/dev/null 2>&1;then
+                    semanage fcontext -a -t system_probe_t $INSTALL_DIR/embedded/bin/system-probe
```

Comment:
what if `semanage` or `restorecon` fail for some reason?

Reply:
You're right, I can do this instead:
```
semanage fcontext -a -t system_probe_t $INSTALL_DIR/embedded/bin/system-probe && restorecon -v $INSTALL_DIR/embedded/bin/system-probe
if [ "$?" !=  "0" ]; then
  echo "Could not relabel the system-probe binary with type system_probe_t."
fi
```

No idea what instructions to give in that case though, because I don't know these commands' failure modes (except the obvious "the file does not exist" (which is already checked above), or "the type does not exist" which should not happen given that the policy needs to be loaded to be able to reach this section).